samples/openapi3/client/petstore/dart-dio/oneof_polymorphism_and_inheritance/pubspec.yaml

```diff
@@ -2,6 +2,7 @@ name: openapi
 version: 1.0.0
 description: OpenAPI API client
 homepage: homepage
+repository: repository
```

Comment:
so this will be always added to every pubSpec? even when I'm never planning to publish in any pubrepo?

Reply:
Should only be added if `repository` is added ... 🤔 
https://github.com/OpenAPITools/openapi-generator/pull/15232/files#diff-539b2a323e510d12ce01710e4cd93ce8ad4bad735afb3035359127fc1644dc78R5

Documentation: https://github.com/janl/mustache.js#false-values-or-empty-lists

Reply:
Now the samples are more representative of what the default behavior is :)